pkg/sqlite/image.go

```diff
@@ -141,6 +143,7 @@ func (i *ImageLoader) loadManifests(manifests string, annotationsFile *registry.
 		return fmt.Errorf("error getting csv from bundle %s: %s", bundle.Name, err)
 	}
 
+	// This isn't necessarily right anymore, because the head of the channels may not be what we are adding
 	packageManifest, err := translateAnnotationsIntoPackage(annotationsFile, bcsv)
```

Comment:
this just shouldn't be called here, right? it should only be called after we know we're adding the head of a channel

Reply:
I suppose we could make that condition and then build a different object to pass into a totally different function, but I was attempting to make as few changes as possible here. Right now the concept of the synthetic manifest is used to define the set of channel metadata anyway and not really a 1:1 translation of "this manifest is what we are judiciously applying to the cluster".

I think if we wanted to do that it would require a slightly more extensive refactor to what we do for adding bundle images now